### ISSUE TITLE
When i use K.clear_session() , the kernal died

### ISSUE THREAD

**Issue by gallonskj** [keras-team/keras#13323]
this is my code 
`def neural_test(p,q,position):
    if(i%10==0):
        K.clear_session() 
    x_train, x_test, y_train, y_test = train_test_split(p, q, test_size=0.4,random_state=1)
    model = Sequential()
    model.add(Dense(20, input_dim=1))
    model.add(Activation('relu'))
    model.add(Dense(1, input_dim=20))
    model.add(Activation('sigmoid'))
    model.compile(optimizer='adam',
              loss='binary_crossentropy',
              metrics=['accuracy'])
    his = model.fit(x_train, y_train, epochs=10, verbose=0,batch_size=16)
    print('last epoch loss: {}   acc: {}'.format(his.history['loss'][-1],his.history['acc'][-1]))
    score = model.evaluate(x_test, y_test, batch_size=16) 
    print(score)`
i want to release GPU memory,but when i use K.clear_session ,the kernal died
